mlflow/server/js/src/experiment-tracking/components/runs-compare/config/RunsCompareConfigureScatterChart.tsx

```diff
@@ -119,7 +119,7 @@ export const RunsCompareConfigureScatterChart = ({
       <RunsCompareRunNumberSelect
         value={state.runsCountToCompare}
         onChange={updateVisibleRunCount}
-        options={runsCompareRunCountDefaultOptions}
```

Comment:
From the code perspective, we might want to do something like this:

```
runsCompareRunCountDefaultOptions.push(
{
    value: 100,
    label: (
      <FormattedMessage
        defaultMessage='100'
        description='Label for 100 first runs visible in run count selector within runs compare configuration modal'
      />
    ),
  },
{
    value: 250,
    label: (
      <FormattedMessage
        defaultMessage='250'
        description='Label for 250 first runs visible in run count selector within runs compare configuration modal'
      />
    ),
  },
{
    value: 500,
    label: (
      <FormattedMessage
        defaultMessage='500'
        description='Label for 500 first runs visible in run count selector within runs compare configuration modal'
      />
    ),
  },
)
```

cc @hubertzub-db ^^

Reply:
Great idea - done!